client/src/components/Dashboard.js

```diff
@@ -166,7 +188,13 @@ class Dashboard extends Component {
           >
             Add document
           </Button>
-        ) : null}
+        ) : (
+          <Checkbox
+            toggle
+            label="Show documents I can translate"
```

Comment:
Change the label to: "Only show documents I can translate"

Reply:
done